File: decompiled/sources/com/google/android/gms/fido/u2f/U2fPendingIntent.java
package com.google.android.gms.fido.u2f;

import android.app.Activity;
import android.content.IntentSender;

public interface U2fPendingIntent {
    boolean hasPendingIntent();

    void launchPendingIntent(Activity activity, int i) throws IntentSender.SendIntentException;
}

File: decompiled/sources/com/google/android/gms/fido/u2f/U2fPendingIntent.kt
package com.google.android.gms.fido.u2f

import android.app.Activity
import android.content.IntentSender

interface U2fPendingIntent {
    fun hasPendingIntent(): Boolean

    @Throws(IntentSender.SendIntentException::class)
    fun launchPendingIntent(activity: Activity, requestCode: Int)
}
